src/EntityFramework.SqlServer/SqlProviderServices.cs

```diff
@@ -115,23 +117,35 @@ public static SqlProviderServices Instance
         public static string SqlServerTypesAssemblyName { get; set; }
 
         /// <summary>
-        /// Set this flag to false to prevent <see cref="decimal" /> values from being truncated to
-        /// the scale (number of decimal places) defined for the column. The default value is true,
-        /// indicating that decimal values will be truncated, in order to prevent breaking existing
-        /// applications that depend on this behavior.
-        /// </summary>
-        /// <remarks>
-        /// With this flag set to true <see cref="SqlParameter" /> objects are created with their Scale
-        /// properties set. When this flag is set to false then the Scale properties are not set, meaning
-        /// that the truncation behavior of SqlParameter is avoided.
-        /// </remarks>
+        /// Get or sets a value indicating whether <see cref="decimal" /> parameter values are truncated to 
+        /// the scale (number of decimal places) defined for their corresponding columns when they are sent 
+        /// to the database. A value of <c>true</c> indicates that <see cref="SqlParameter" /> objects 
+        /// created for <see cref="decimal" /> columns will have their <see cref="SqlParameter.Scale" /> 
+        /// properties set, which will cause the parameter values to be truncated. If set to <c>false</c> 
+        /// then the <see cref="SqlParameter.Scale" /> properties will not be set, avoiding the truncation 
+        /// behavior of <see cref="SqlParameter" /> and allowing SQL Server to round values if necessary. 
+        /// The default value is <c>true</c> to prevent breaking existing applications that depend on this 
+        /// behavior. 
         public static bool TruncateDecimalsToScale
         {
             get { return _truncateDecimalsToScale; }
             set { _truncateDecimalsToScale = value; }
         }
 
         /// <summary>
+        /// Gets or sets a value indicating whether to use the SCOPE_IDENTITY() function to retrieve values 
+        /// generated by the database for numeric columns during an INSERT operation. The default value of 
+        /// <c>true</c> is recommended and can provide better performance if all numeric values are generated using 
+        /// IDENTITY columns. If set to <c>false</c>, an OUTPUT clause will be used instead. An OUTPUT clauses 
```

Comment:
Just noticed a typo (presumably my typo 😄), "An OUTPUT clauses..." should be "An OUTPUT clause..."